src/pages/Confirm/Confirm.layout.jsx

```diff
@@ -11,10 +11,10 @@ const Confirm = ({ headerTitle, headerSubTitle, children, RatingModal, FinalModa
     <Header>
       <BackButton />
       <HeaderContainer>
-        <Text as="h1" color="secondary" xlarge>
+        <Text as="h1" color="third" weight700 xlg bold>
```

Comment:
Really is working "weight700"? I think doesn't look good have attributes with numbers. Maybe looks better if you change the name for a reference, for example: weight 700 can be called "semibold", while weight 800 can be called "bold", and weight 400 can be called "normal", what do you think?

```suggestion
        <Text as="h1" color="third" weight700 xlg bold>
```